lib/Cmts.ml

```diff
@@ -562,8 +562,22 @@ module Verbatim = struct
     $ wrap "(*" "*)" @@ str s
 end
 
-let fmt_cmt (cmt : Cmt.t) ~wrap:wrap_comments ~ocp_indent_compat ~fmt_code
-    pos =
+module Ocp_indent_compat = struct
+  let fmt ~fmt_code conf (cmt : Cmt.t) (pos : Cmt.pos) =
+    let parsed = Docstring.parse ~loc:cmt.loc cmt.txt in
+    (* Disable warnings when parsing fails *)
+    let quiet = Conf_t.Elt.make true `Default in
+    let conf = {conf with Conf.opr_opts= {conf.Conf.opr_opts with quiet}} in
+    let doc = Fmt_odoc.fmt_parsed conf ~fmt_code ~input:cmt.txt parsed in
+    let open Fmt in
+    fmt_if_k
+      (Poly.(pos = After) && String.contains cmt.txt '\n')
+      (break_unless_newline 1000 0)
+    $ wrap "(*" "*)" @@ doc
+end
+
+let fmt_cmt conf (cmt : Cmt.t) ~wrap:wrap_comments ~ocp_indent_compat
```

Comment:
`~wrap` and `~ocp_indent_compat` are already contained in `conf`.

Reply:
done